tools/run_tests/dockerize/build_interop_image.sh

```diff
@@ -75,6 +75,22 @@ git branch -f jenkins-docker
 
 CONTAINER_NAME="build_${BASE_NAME}_$(uuidgen)"
 
+# build_interop.sh could be missing from old git tree and we will have issue unless we find the proper
+# build_interop.sh to use in such case.
+if ! [ -e $GRPC_ROOT/tools/dockerfile/interoptest/$BASE_NAME/build_interop.sh ]; then
```

Comment:
I get that but that's an inconsistency. if you are using grpc_interop_cxx_debian8/Dockerfile  from head, you should also use grpc_interop_cxx_debian8/build_interop.sh from HEAD. Or you should use Dockerfile and build_interop.sh from the old revision. But combining one of them from head seems as a hack.

Can we adding unnessary complexity to build_interop_image.sh? Interop test building is already complicated enough. If you really need to override the build_interop.sh file location, can you do that in some other way? (e.g. env variable set by the matrix script or sth in that sense)